### DISCUSSION TITLE
Show humidity, not luxes :)

### DISCUSSION THREAD

**Discussion by dimkagithub** [merdok/homebridge-miot#455] [Ideas]
Hi!
It would be great if the Xiaomi Mi Temperature and Humidity Sensor 2 showed humidity, not luxes.
Temperature is ok, battery is ok, but humidity is luxes.
<img width="509" alt="Снимок экрана 2023-03-17 в 23 57 49" src="https://user-images.githubusercontent.com/71010307/225994697-77fafad1-e1d9-48d0-99cd-0ae531b5018c.png">

That's how it is here:
<img width="355" alt="Снимок экрана 2023-03-18 в 00 02 08" src="https://user-images.githubusercontent.com/71010307/225997975-cb94e6e7-36cb-465c-b3d8-7483b5596e81.png">

Is it possible?

**Comment by merdok** [comment]
Do you use a property monitor for that? I will need to add proper humidity and temperature sensor support first in order for it to work.
What device models do you use?

**Reply by dimkagithub** [reply]
At the moment I use property monitor.

Model is **miaomiaoce.sensor_ht.t2**

```
{
    "name": "Thermometer",
    "ip": "192.168.1.25",
    "token": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
    "deviceId": "xxx.x.xxxxxxxxxxxxxxx",
    "model": "miaomiaoce.sensor_ht.t2",
    "pollingInterval": 10,
    "deepDebugLog": false,
    "micloud": {
        "username": "xxxxxxxxxxxxxx@xxxxxxxx.xxx",
        "password": "xxxxxxxxxxxxxx",
        "country": "xx",
        "forceMiCloud": true
    },
    "buzzerControl": true,
    "ledControl": true,
    "childLockControl": true,
    "modeControl": true,
    "propertyMonitor": [
        {
            "property": "2.1",
            "name": "Themperature"
        },
        {
            "property": "2.2",
            "name": "Humidity"
        }
    ]
}
```

**Reply by merdok** [reply]
Yeah that is the issue. Property monitor tries to detect the property type and selects an accessory to display it, when it cannot detect the type then a mix sensor is used because it can display any numeric value.
I would need to add humidity and temperature accessories to properly support such sensors.
Could you please open an issue with the device models for that so I do not forget about it?

**Reply by dimkagithub** [reply]
Of course, I can open an issue )
Thanks!

Issue is opened.

**Reply by merdok** [reply]
Thanks!
